pkg/apis/kops/cluster.go

```diff
@@ -133,8 +133,8 @@ type ClusterSpec struct {
 	IsolateMasters *bool `json:"isolateMasters,omitempty"`
 	// UpdatePolicy determines the policy for applying upgrades automatically.
 	// Valid values:
-	//   'external' do not apply updates automatically - they are applied manually or by an external system
-	//   missing: default policy (currently OS security upgrades that do not require a reboot)
```

Comment:
Note that this avoidance of reboots was and is not true for Flatcar Container Linux. It _only_ installs updates by way of rebooting. It downloads and verifies them ahead of time, but only pivots to using the new files upon reboot.